engine.py

```diff
@@ -755,6 +767,35 @@ def show_panel(self, panel_id, title, bundle, widget_class, *args, **kwargs):
         # resolved by looking at the stream of event and force triggering refreshes at the
         # right locations
         #
-        tk_maya.install_callbacks(panel_id, widget_id)
+        tk_maya.install_callbacks(maya_panel_name, widget_id)
 
         return widget_instance
+
+    def close_dialogs_and_panels(self):
```

Comment:
I am thinking `close_windows` may be a better name in case we decide to add more types of uis in the future (beyond dialogs and panels). With windows meaning both panels and dialogs. I am not in love with `windows` - thought about `close_uis` but that sounded even worse. What do you think?